doc-content/enterprise-only/admin-and-config/clean-log-proc.adoc

```diff
@@ -0,0 +1,63 @@
+[id='clean-log-proc']
+= Removing logs from the database
+
+Use `LogCleanupCommand` executor command to clean up the data that is using the database space. The `LogCleanupCommand` consists of logic to automatically clean up all or selected data.
```

Comment:
We can use "**which**" instead of "**that**".

Use `LogCleanupCommand` executor command to clean up the data **which** is using the database space.

Reply:
Done